andela/Ah-frontend-guardians
# #162685619 Update TravisCI to run tests

#### What does this PR do?
Update TravisCI to run tests

#### Description of Task to be completed?
The task involves updating TravisCI to run tests
* add a .travis.yml file to the root folder

#### How should this be manually tested?
* Manually check out for the .travis.yml file in the root folder of the project.
* When you look at the review requested session towards the bottom of this page, you will notice that Travis is among the checks that have to be performed. See screenshot below

![image](https://user-images.githubusercontent.com/41106626/52622700-88c6c200-2ebb-11e9-8e2c-a95e57c7783b.png)



#### What are the relevant pivotal tracker stories?
[#162685619](https://www.pivotaltracker.com/story/show/162685619)
